testcafeRunner.js

```diff
@@ -17,7 +17,7 @@ const browserToRun = browserFromArgs.length > 0 ? browserFromArgs : 'chrome:head
 const testFromArgs = process.argv.slice(3, 4);
 const testsToRun = testFromArgs ? `**/*${testFromArgs}*/ui.test.js` : '**/*ui.test.js';
 
-let concurrency = 4;
+let concurrency = 1;
```

Comment:
Next PR whoever gets in there first. Lets take this out. The concurrency doesn't work with the nock checks

Reply:
take out completely?